environments/dev/dev.tfvars

```diff
@@ -17,6 +17,8 @@ ssl_policy = {
 key_vault_subscription        = "867a878b-cb68-4de5-9741-361ac9e178b6"
 hub_app_gw_private_ip_address = ["10.11.72.233"]
 apim_appgw_backend_pool_fqdns = ["firewall-nonprodi-palo-sdsapimgmtdev.uksouth.cloudapp.azure.com"]
+min_capacity                  = 1
+max_capacity                  = 2
```

Comment:
capacity for what?

Reply:
Application gateway instances capacity, default is 2 and 10, let me change the names of variables, one sec